docs/references/javascript/clerk/clerk.mdx

```diff
@@ -188,14 +188,47 @@ function authenticateWithCoinbaseWallet({
   - `signUpContinueUrl?`
   - `string | undefined`
 
-  The location to continue navigation to in the sign up process if data is missing.
+  The URL to navigate to in the sign-up process if data is missing.
 
   ---
 
   - `customNavigate?`
   - `((to: string) => Promise<unknown> | unknown) | undefined`
 
-  Allows you to define a custom navigation function
+  Allows you to define a custom function to handle sign-up and sign-in navigation
+</Properties>
+
+### `authenticateWithOKXWallet()`
+
+```typescript
+function authenticateWithOKXWallet({
+  redirectUrl,
+  signUpContinueUrl,
+  customNavigate,
+}?: AuthenticateWithOKXWalletParams): Promise<void>
+```
+
+#### `AuthenticateWithOKXWalletParams`
+
+<Properties>
+  - `redirectUrl?`
+  - `string | undefined`
+
+  Full URL or path to navigate to after a successful sign-in or sign-up.
+
+  ---
+
+  - `signUpContinueUrl?`
+  - `string | undefined`
+
+  The URL to navigate to in the sign-up process if data is missing.
+
+  ---
+
+  - `customNavigate?`
+  - `((to: string) => Promise<unknown> | unknown) | undefined`
+
+  Allows you to define a custom function to handle sign-up and sign-in navigation
 </Properties>
```

Comment:
looks like `unsafeMetadata` and `legalAccepted` are missing

<img width="491" alt="Screenshot 2024-12-06 at 14 03 48" src="https://github.com/user-attachments/assets/82435680-d80b-4d75-997d-70c71fed29ba">

Reply:
handled here for ya [4f8bdb8](https://github.com/clerk/clerk-docs/pull/1768/commits/4f8bdb810f2309f05eb3162697481189df4c595d)